classes/UpgradeSelfCheck.php

```diff
@@ -31,6 +31,11 @@
 
 class UpgradeSelfCheck
 {
+    /**
+     * Recommended PHP Version. If below, display a notice.
+     */
+    const RECOMMENDED_PHP_VERSION = 70000;
```

Comment:
```suggestion
    const RECOMMENDED_PHP_VERSION = 71000;
```
Php 7.0 is already outdated :/ (http://php.net/supported-versions.php)

Reply:
I know, but only PHP 5.6 could be shortly drop from our compatibility list. I chose to display this message for these people first.

This can be discussed with @PrestaShop/prestashop-core-developers